src/ray/rpc/server_call.h

```diff
@@ -29,6 +29,13 @@
 namespace ray {
 namespace rpc {
 
+// Authentication type of ServerCall.
+enum class AuthType {
+  NO_AUTH,      // Do not authenticate (accept all).
+  LAZY_AUTH,    // Accept missing token, but reject wrong token.
+  STRICT_AUTH,  // Reject missing token and wrong token.
```

Comment:
I think this is dead code at least for now.

Reply:
removed, we can add it back when we use it